server/handler/CheckHandler.js

```diff
@@ -122,7 +122,7 @@ export class CheckHandler {
    */
   async onEnableCheck(user, repository, type) {
     const repo = repository.get('json')
-    const types = [type, ...repository.checks.map(c => c.type)]
+    const types = Array.from(new Set([type, ...repository.checks.map(c => c.type)]))
```

Comment:
I had an error from GitHub at some point, that multiple types where being sent. This just removes possible duplicates.